src/pal/inc/pal.h

```diff
@@ -2410,6 +2410,14 @@ GetThreadTimes(
         OUT LPFILETIME lpKernelTime,
         OUT LPFILETIME lpUserTime);
 
+PALIMPORT
+HRESULT
+PALAPI
+SetThreadName(
```

Comment:
Note that the result is ignored by the calling code anyway. Since this is just a diagnostic aid API, it may be best to just make a `PAL_` function that returns void.